config/locales/en.yml

```diff
@@ -990,6 +990,7 @@ en:
       view_strikes: View past strikes against your account
     too_fast: Form submitted too fast, try again.
     use_security_key: Use security key
+    username_validity: Must contain only a-Z, 0-9, or _
```

Comment:
I am not sure how user-friendly that text is. Maybe “Can only contain letters, numbers and underscores (_)”?